### DISCUSSION TITLE
Welcome to the Varanasi internship 2023

### DISCUSSION THREAD

**Discussion by eeholmes** [UW-Upwelling-Project/Summer-2023#1] [Announcements]
Hi Minh and Jiarui,

I am looking forward to working with you this summer with the Varanasi internship! There are 2 interns on the upwelling project so you will be able to work together. We are also arranging for an office at SAFS for the Varanasi interns so they can work in person on campus.  Minh may be working remote this summer, but there will be 2 other Varanasi interns in the office.

We will use GitHub for discussions and tracking work:
https://github.com/UW-Upwelling-Project/Summer-2023

You can look at the prior summers' discussion tabs to get a sense of past years.

Below is lots of info for the summer. We can go over this when we have our first in person meeting. I suggest we do this in May (see below).

Here are some dates:

Sometime in May. In person meeting to meet and talk about the project. No work at all required. This is a chance to ask questions and get an overview. Let me know days that are generally good for you when you are on campus and I'll set up a time.

June 12th: Start of the internship. You can start earlier but this is a summer internship so definitely do not feel you need to do that!

OceanHackWeek Aug 7-11. Mark your calendars as we will be participating in this. It is a hybrid event. So no worries if you need to be remote that week, but if you are in town, it will be a fun in-person event.

Preliminary work. 

Note, I don't expect you to work on the internship while school is in session, but both of you asked about background work.

Read/skim the reports from the last 2 years:
2021: https://github.com/UW-Upwelling-Project/Summer-2021/discussions/45

2022: https://github.com/UW-Upwelling-Project/Summer-2022/blob/main/Analysis_Upwelling_changes_using_Wavelet_Analysis-2-compressed.pdf

Learning how to access remote-sensing data with Python:

The first goal will be for you to get up to speed on accessing remote-sensing data in the cloud. You'll be working through these tutorials. I am not sure you will be able to work with these on your laptop. Maybe? But I am going to try to get us set up with a JupyterHub in the cloud. TBD.

https://nasa-openscapes.github.io/earthdata-cloud-cookbook/
https://podaac.github.io/2022-SWOT-Ocean-Cloud-Workshop/

**Comment by minhphan03** [comment]
@eeholmes I just got an email from OceanHackWeek that I was not offered a spot this year. Is there any chance we can do something else instead, or appeal the decision?

**Comment by eeholmes** [comment]
Hmm, I'll check in with the OHW organizers.
